content/concepts/concepts.md

```diff
@@ -0,0 +1,129 @@
+
+In webpack we define _entry points_ using the `entry` property in our [webpack configuration object](./configuration). 
+
+The simplest example is seen below: 
+
+**webpack.config.js**
+```
+  module.exports = config; 
+
+  const config = {
+    entry: './path/to/my/entry/file.js' 
+  }
+
+```
+
+There are multiple ways to declare your `entry` property that are specific to your applications needs. 
+
+[**Learn more!**](./entry-points)
+
+## Output
+Once you've bundled all of your assets together, we still need to tell webpack **where** to bundle our application. The webpack `output` property describes to webpack **how to treat bundled code**. 
+
+**webpack.config.js**
+```
+  module.exports = config; 
+
+  const config = {
+    entry: './path/to/my/entry/file.js',
+    output: {
+      filename: 'my-first-webpack.bundle.js',
+      path: __dirname + 'dist/'
+    }
+  }
+
+```
+
+In the example above, through the `output.filename` and `output.path` properties we are describing to webpack the name of our bundle, and where we want it to be emitted to.
+
+T> You may see the term **emitted** or **emit** used throughout our documentation and [plugin API](../api/plugins). This is a fancy term for "produced or discharged". 
+
+The `output` property has [many more configurable features](../api/configuration), but lets spend some time understanding some of the most common use cases for the `output` property. 
+
+[**Learn more!**](./output)
+
+
+## Loaders
+The goal is to have all of your assets in your project to be **webpack's** concern and not the browser. (This doesn't mean that they all have to be bundled together). webpack treats [every file (.css, .html, .scss, .jpg, etc.) as a module](./everything-is-a-module). However, webpack **only understands JavaScript**.
+
+**Loaders tell webpack _how to treat these files as modules_ as they are added to your dependency graph.**
+
+At a high level, they have two purposes in your webpack config. 
+1. Identify what files should be transformed by a certain loader. (`test` property)
+2. Transform that file so that it can be added to your dependency graph (and eventually your bundle). (`loader` property)
+
+**webpack.config.js**
+```
+  module.exports = config; 
+
+  const config = {
+    entry: './path/to/my/entry/file.js',
+    output: {
+      filename: 'my-first-webpack.bundle.js',
+      path: __dirname + 'dist/'
+    },
+    module: {
+      loaders: [
+        {test: /.jsx?$/, loader: 'babel-loader'} 
+      ]
+    }
+  };
+```
+
+In the configuration above we have defined our loader with its two required properties: `test`, and `loader`. It tells webpack's compiler the following: 
+
+```"Hey webpack compiler, when you come across a file with extension '.js' or '.jsx' inside of a require() statement, use the babel-loader to transform it before you bundle it together".```
```

Comment:
Yeah, great point. Maybe that should go to `concepts/modules.md` since it's related to module resolution.